python/ray/job_config.py

```diff
@@ -15,26 +25,17 @@ class JobConfig:
             `CLASSPATH` in Java and `PYTHONPATH` in Python.
     """
 
-    def __init__(
-            self,
-            worker_env=None,
-            num_java_workers_per_process=1,
-            jvm_options=None,
-            code_search_path=None,
-    ):
-        if worker_env is None:
-            self.worker_env = dict()
-        else:
-            self.worker_env = worker_env
+    def __init__(self,
+                 worker_env=None,
+                 num_java_workers_per_process=1,
+                 jvm_options=None,
+                 code_search_path=None,
+                 runtime_env=RuntimeEnv()):
```

Comment:
IIRC in the original spec we had this as a dictionary. Can we use that instead of a class for consistency with runtime_env in other API call sites?

Reply:
Oh, good catch. Yeah let's go with dict for now, we can always add a class as an option

Reply:
Sure. I made it a dict at the beginning, then realized that maybe we want it typed since it's accessing certain field.